src/app/backend/resource/logs/logs.go

```diff
@@ -251,7 +251,7 @@ func ToLogLines(rawLogs string) LogLines {
 	logLines := LogLines{}
 	for _, line := range strings.Split(rawLogs, "\n") {
 		if line != "" {
-			startsWithDate := ('0' <= line[0] && line[0] <= '9') //2017-...
+			startsWithDate := '0' <= line[0] && line[0] <= '9' //2017-...
```

Comment:
It's an only cosmetic change. This is not required.

Reply:
Yeap! this is from a IDE format warning.